fedimint-api/src/lib.rs

```diff
@@ -14,13 +14,18 @@ use thiserror::Error;
 
 use crate::encoding::{Decodable, DecodeError, Encodable};
 
+pub use tiered::Tiered;
+pub use tiered_multi::*;
+
 pub mod config;
 pub mod db;
 pub mod encoding;
 pub mod macros;
 pub mod module;
 pub mod rand;
 pub mod task;
+pub mod tiered;
+pub mod tiered_multi;
```

Comment:
Do both the mod and the use above need to be pub?

Reply:
Probably not, but the fact that they are available both as `fedimint-api::Tiered` and `fedimint-api::tiered::Tiered` doesn't matter much. I think we're going to move it around again anyway.